components/ledger/internal/bootstrap/config.go

```
@@ -53,63 +60,186 @@ type Config struct {
-// both onboarding and transaction modules in a single process.
-// The transaction module is initialized first so its BalancePort (the UseCase)
-// can be passed directly to onboarding for in-process calls.
+// InitServers initializes the unified ledger service.
 func InitServers() (*Service, error) {
 	return InitServersWithOptions(nil)
 }
 
 // InitServersWithOptions initializes the unified ledger service with optional dependency injection.
+// It directly initializes all infrastructure (PG, Mongo, Redis, RabbitMQ) instead of delegating
+// to onboarding/transaction sub-modules.
 func InitServersWithOptions(opts *Options) (*Service, error) {
```

golangci-lint (2.11.3)

[error] cognitive complexity 33 of func `InitServersWithOptions` is high (> 30)

(gocognit)